src/search/landmarks/landmark_graph.cc

```diff
@@ -41,11 +28,15 @@ int LandmarkGraph::get_num_edges() const {
     return total;
 }
 
-LandmarkNode *LandmarkGraph::get_landmark(int i) const {
+const Landmark &LandmarkGraph::get_landmark(int i) const {
```

Comment:
The interface would be cleaner without this method. Now there are two equivalent ways of accessing a landmark:
`graph.get_landmark(i)` and `graph.get_landmark_node(i)->get_landmark()`. From the outside it is unclear if there is a difference between them or if one should be preferred.

Reply:
We removed the possibility to directly access the landmark by the ID of the corresponding node.